modules/_labs/select/react/stories/stories_testing.tsx

```diff
@@ -0,0 +1,136 @@
+/// <reference path="../../../../../typings.d.ts" />
+import * as React from 'react';
+
+import {colors} from '@workday/canvas-kit-react-core';
+import {Button} from '../../../../button/react';
+import FormField from '../../../../form-field/react';
+import Modal from '../../../../modal/react';
+import Select from '../lib/Select';
+
+import {manyOptions, options} from './stories';
+import {controlComponent} from '../../../../../utils/storybook';
+
+export default {
+  title: 'Testing|React/Labs/Select',
+  component: Select,
+};
+
+const Container = ({children, style = {}, ...elemProps}) => {
+  return (
+    <div
+      style={{
+        border: `1px solid ${colors.soap600}`,
+        marginTop: 50,
+        overflow: 'hidden',
+        padding: 20,
+        ...style,
+      }}
+      {...elemProps}
+    >
+      {children}
+    </div>
+  );
+};
+
+const SelectModal = () => {
+  const [open, setOpen] = React.useState(false);
+  const buttonRef = React.useRef<HTMLButtonElement>() as React.RefObject<HTMLButtonElement>; // cast to keep buttonRef happy
+  const openModal = () => {
+    setOpen(true);
+  };
+  const closeModal = () => {
+    setOpen(false);
+    if (buttonRef.current) {
+      buttonRef.current.focus();
```

Comment:
Not sure why this line is here? The Modal should take care of focus changes for you.

Reply:
Not sure, I copied code from an older (pre-stack-manager) Modal story without much thought. This code no longer exists now that we're using `useModal`.